File: Kotlin/Easy/Jack-O'-Lantern Juxtaposition/Jack_O_Lantern.kt
fun main(args: Array<String>) {
    var line = readLine()

    while(line != null)
    {
        var (eyes, nose, mouths) = line.split(' ')

        var product: Int = eyes.toInt() * nose.toInt() * mouths.toInt();

        println(product)

        line = readLine();
    }
}

File: JavaScript/Easy/Jack-O'-Lantern Juxtaposition/Jack_O_Lantern.js
const readLine = require("readline");

const rl = readLine.createInterface({input: process.stdin, output:process.stdout});

rl.on('line', (line) => {
    var input = line.split(' ');
    var eyes = parseInt(input[0])
    var nose = parseInt(input[1])
    var mouths = parseInt(input[2])

    var result = eyes*nose*mouths;

    console.log(result);
});